src/tls.rs

```diff
@@ -149,6 +149,28 @@ impl Context {
         }
     }
 
+    pub fn load_verify_locations(
```

Comment:
It would be better to also split this into two (`_from_file` and `_from_directory`) variants, so we don't have to deal with the `Option`s.

Reply:
Done